docs/en/faq/how-kotlin-jdsl-is-different-from-jooq-and-querydsl.md

```diff
@@ -0,0 +1,34 @@
+# How is Kotlin JDSL different from jOOQ and QueryDSL?
+
+Unlike jOOq and QueryDSL, Kotlin JDSL does not generate a `Metadata Model (code generation) for writing queries.
+
+If you use code generation, you'll have to do the following.
+
+```
+Modify tables / entities →
+Compilation errors due to code and metacode mismatch →
+Temporarily change all code associated with the modified code, e.g. by commenting it out, to make it compile.
+Regenerate code
+Remove the temporary code that was processed to pass compilation and replace it with working code.
+```
+
+In addition, JPQL uses a string-based query writing method, so it can pass compilation but may cause errors when running the web application.
+
+However, Line-JDSL allows you to write queries based on pure code rather than using string-based writing and code generation.
```

Comment:
Why use Line-JDSL instead of Kotlin JDSL?

Reply:
Apparently it was created by `Line`, so I was a little confused about its official name. I'll fix that.